layouts/shortcodes/glossary.html

```diff
@@ -0,0 +1,16 @@
+{{ range sort $.Site.Data.glossary.entries "term" }}
+    <div>
+        <ul>
+            <li><a href="#{{ lower .term }}" class="">{{ .term }}</a></li>
+        </ul>
+    </div>
+{{ end }}
+    
+{{ range sort $.Site.Data.glossary.entries "term" }}
+    <div class="flex-1 mt-8">
+        <h2 class="mt-2 mb-0">{{ .term }} </h2>
+            <p>{{ .description }}</p>
+                <ul class="mb-4">
+                    <li>Read More: <a href="{{ .link }}" class="">{{ .term }}</a></li>
```

Comment:
I wonder if we should use `relref` here, which will ensure the pages we're linking to exist. (This is helpful when moving pages around / restructuring the site).

```suggestion
                    <li>Read More: <a href="{{ relref . .link }}" class="">{{ .term }}</a></li>
```